packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -32,10 +32,10 @@ test.describe('modal', () => {
         });
 
         // Act
-        const modal = await modalPage.isModalVisible();
+        const modal = await modalComponent.isModalVisible();
```

Comment:
Could you we change this to `isModalVisible`?

Reply:
Or change to `expect(await modalComponent.isModalVisible()).toBe(true);` so it's consistent with other tests.